src/app/lib/utilities/getEnvConfig/index.test.ts

```diff
@@ -1,46 +1,43 @@
 import { getEnvConfig } from '.';
+import * as onClient from '../onClient';
 
-let windowSpy: jest.SpyInstance<Window | undefined, []>;
+const onClientSpy = jest.spyOn(onClient, 'default');
 
 describe('getEnvConfig', () => {
   const originalProcessEnv = process.env;
 
   beforeEach(() => {
+    onClientSpy.mockImplementation(() => false);
     process.env = originalProcessEnv;
-    windowSpy = jest.spyOn(window, 'window', 'get');
   });
 
   afterEach(() => {
-    windowSpy.mockRestore();
+    jest.clearAllMocks();
+    onClientSpy.mockClear();
+    // @ts-expect-error The operand of a 'delete' operator must be optional.ts(2790)
+    delete window.SIMORGH_ENV_VARS;
```

Comment:
Slight nitpick/something I don't understand. 

These seem to be only used in the last test `it('client side - should return values from "getEnvConfig"`. Do we need this as it's only relevant to the final test, or is this to prevent it leaking into other unit tests? If so, could it just be added to after the regression expects in the test, rather than in afterEach()

Reply:
Yeah I think I was paranoid about leakage into other tests but you make a valid point. Let me move the deletion next to the actual test that needs it, just in case.